src/native/managed/cdacreader/src/Data/EEClass.cs

```diff
@@ -15,9 +15,24 @@ public EEClass(Target target, TargetPointer address)
         MethodTable = target.ReadPointer(address + (ulong)type.Fields[nameof(MethodTable)].Offset);
         NumMethods = target.Read<ushort>(address + (ulong)type.Fields[nameof(NumMethods)].Offset);
         CorTypeAttr = target.Read<uint>(address + (ulong)type.Fields[nameof(CorTypeAttr)].Offset);
+        InternalCorElementType = target.Read<byte>(address + (ulong)type.Fields[nameof(InternalCorElementType)].Offset);
     }
 
     public TargetPointer MethodTable { get; init; }
     public ushort NumMethods { get; init; }
     public uint CorTypeAttr { get; init; }
+    public byte InternalCorElementType { get; init; }
```

Comment:
Please doc what makes this "Internal", that is generally an important distinction.